src/nep.cpp

```diff
@@ -1243,7 +1243,7 @@ void find_descriptor_for_lammps(
 
     double F = 0.0, Fp[MAX_DIM] = {0.0}, latent_space[MAX_NEURON] = {0.0};
     apply_ann_one_layer(
-      annmb.dim, annmb.num_neurons1, annmb.w0, annmb.b0, annmb.w1, annmb.b1, q, F, Fp,
+      annmb.dim, annmb.num_neurons1, annmb.w0[t1], annmb.b0[t1], annmb.w1[t1], annmb.b1, q, F, Fp,
```

Comment:
Why is the variable name for the different types `t1` instead of, e.g., `type`?

Reply:
just to make it shorter :-)
In the code:
`t1`  = type of atom `n1`
`t2` = type of aotm `n2`

Reply:
It was defined here:
```
int t1 = g_type[n1] - 1; // from LAMMPS to NEP convention
```

Reply:
Alright :+1: